src/contact-summary/lib.js

```diff
@@ -4,11 +4,16 @@ function isReportValid(report) {
   return report && !(report.errors && report.errors.length);
 }
 
+function getType(contact) {
+  return contact.type === 'contact' ? contact.contact_type : contact.type;
+}
+
 var result = {
   cards: [],
   fields: fields.filter(function(f) {
-        if(f.appliesToType === contact.type ||
-            (f.appliesToType.charAt(0) === '!' && f.appliesToType.slice(1) !== contact.type)) {
+        var type = getType(contact);
+        if(f.appliesToType === type ||
```

Comment:
This may be an issue for another ticket but our docs say this should support an array but the code here causes an error when an array is provided as the `appliesToType`

Reply:
Also it seems that the cards appliesToType code should also accept arrays but doesn't. Since this isn't apart of this ticket I'm going to open a new one to fix that.

Reply:
Good spotting. Thanks!